monai/networks/nets/ahnet.py

```diff
@@ -458,6 +461,10 @@ def __init__(
                 m.weight.data.fill_(1)
                 m.bias.data.zero_()
 
+        if pretrained:
+            net2d = FCN(pretrained=True)
```

Comment:
should we use MFCN here for more generic interface? need to add the `progress` option.  @yiheng-wang-nv please  review this PR

Reply:
Hi @wyli , here we use the pretrained weights from ResNet50 which is in FCN, MCFCN also uses the same pretrained weights. You can take look at this test example: https://github.com/Project-MONAI/MONAI/blob/master/tests/test_ahnet.py#L98

Thanks.